src/main/java/vfyjxf/bettercrashes/BetterCrashesConfig.java

```diff
@@ -19,8 +21,21 @@ public static void init(File file) {
     public static void syncConfig() {
         config.setCategoryComment(GENERAL, "General config");
 
-        isGTNH = config.get(GENERAL, "isGTNH", false, "Set to false if you're playing outside of GTNH")
-                .getBoolean();
+        isGTNH = config.getBoolean("isGTNH", GENERAL, false, "Set to false if you're playing outside of GTNH");
```

Comment:
Can it be rather detected by looking if coremod is present?

Reply:
sounds fair